spec/requests/api/v1/trainer/pokemons_request_spec.rb

```diff
@@ -166,21 +166,23 @@
       expect(error_response[:errors]).to include("unexpected token at 'Not Found'")
     end
 
-    it "Returns error from invalid patch params" do
+    xit "Returns error from invalid patch params" do
       mew_id = @mew.id
-      mew_before_energy = @mew.energy
-      mew_before_happiness = @mew.happiness
-      poke_params = {energy: "potato", fake_attribute: 75}
+      # mew_before_energy = @mew.energy
+      # mew_before_happiness = @mew.happiness
+      poke_params = { pokemon: { energy: "potato", fake_attribute: 75 } } 
 
       patch "/api/v1/trainers/#{@trainer1.id}/pokemons/#{@mew.id}", params: poke_params
-
-      expect(response).to be_successful
-      error_response = JSON.parse(response.body, symbolize_names: true)
+      # expect(response).to have_http_status(:unprocessable_entity)
       
+      require 'pry'; binding.pry
```

Comment:
Good that were testing with pry, this should be removed before submission, but im glad we're putting them to good use